lib/iris/coords.py

```diff
@@ -516,13 +516,26 @@ def copy(self, points=None, bounds=None):
         .. note:: If the points argument is specified and bounds are not, the
                   resulting coordinate will have no bounds.
 
+        .. deprecated:: 1.12
+
+            By default the new coordinate's `points` and `bounds` will
```

Comment:
I think this explains it from slightly the wrong angle : see the `Cube.__getitem__` comment...